### PR TITLE
Followed on my timeline

### PR DESCRIPTION
**Pull request by Faberen**

### CODE REVIEW

File: src/Chirp.Infrastructure/CheepRepository.cs [hunk 1]
```diff
@@ -222,4 +222,166 @@ public async Task IncreaseLikeAttributeInCheep(Guid cheepId)
...
+                    CheepId = CheepDTO.CheepId,
+                    AuthorName = CheepDTO.Author.Name,
+                    Message = CheepDTO.Text,
+                    Likes = CheepDTO.Likes,
+                    Timestamp = CheepDTO.TimeStamp
+                }
+            ).ToList();
+            cheepsToReturn.AddRange(cheepsFromAuthors);
+        }
+        //Change later
```
Comment (AlbinoLoaf) [inline src/Chirp.Infrastructure/CheepRepository.cs:260]: Remove comment

**Review by AlbinoLoaf — CHANGES_REQUESTED** [review]
Add

**Review by annahj1006 — APPROVED** [review]
Some of them are quite large methods, perhabs add some more description for readability? Also one comment on line 334 is not complete?
Otherwise really nice work!

File: src/Chirp.Infrastructure/CheepRepository.cs [hunk 2]
```diff
@@ -222,4 +222,166 @@ public async Task IncreaseLikeAttributeInCheep(Guid cheepId)
...
+//         {
+//             return cheeps.GetRange(0, 32);
+//         }
+//         else
+//         {
+//             int endIndex = Math.Min((int)page + 32, (int)cheeps.Count);
+//             return cheeps.GetRange((int)page, endIndex - (int)(page));
+//         }
+
+//     }
```
[the remark this answers is not in the document]
Comment (Lukan707) [reply src/Chirp.Infrastructure/CheepRepository.cs]: This has been resolved in commit 18e8b19.

@Marius it is okay to solve theese minimum effort fixes yourself, and just comment. I can also see that you are a co-author of this :)

File: src/Chirp.Infrastructure/CheepRepository.cs [hunk 3]
```diff
@@ -222,4 +222,166 @@ public async Task IncreaseLikeAttributeInCheep(Guid cheepId)
...
+            AuthorName = joinResult.Author.Name,
+            Message = joinResult.Cheep.Text,
+            Likes = joinResult.Cheep.Likes,
+            Timestamp = joinResult.Cheep.TimeStamp
+        });
+        cheepsToReturn.AddRange(cheepsDTO);
+
+        return cheepsToReturn;
+    }
+
```
[the remark this answers is not in the document]
Comment (Lukan707) [reply src/Chirp.Infrastructure/CheepRepository.cs:289]: I belive this is evident from the code.

However I have commented in commit 18e8b19, what the two functions are used for :)

**Comment by Lukan707** [comment]
> Some of them are quite large methods, perhabs add some more description for readability? Also one comment on line 334 is not complete? Otherwise really nice work!

I am unable to find the comment you are reffering to. In which file is the incomplete comment placed in? 

I have added an issue regarding the comments on large methods: #247

**Review by SkarpKat — APPROVED** [review]

**Review by Lukan707 — APPROVED** [review]
Nice work @Faberen and @AlbinoLoaf

**Review by AlbinoLoaf — APPROVED** [review]

**Pull request merged**